src/App.vue

```diff
@@ -35,17 +44,24 @@ export default {
   data() {
     return {
       comment: '',
-      lists: []
+      lists: [],
     }
   },
   methods: {
     addTodo: function(){
-      this.lists.push(this.comment)
+      const todo = {
+        comment: this.comment,
+        state: true,
+      }
+      this.lists.push(todo)
     },
     deleteList: function(i){
       this.lists.splice(i, 1)
     },
+    changeState: function(i){
```

Comment:
functionを省略する記法もありますので、ぜひ使っていきましょう。
また`{}`の両外側にはスペースをあけるようにしましょう。
```suggestion
    changeState(i) {
```